src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -0,0 +1,184 @@
+import _ from "lodash"
+import SharedService from "../services/SharedService"
+import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import { pointSub } from "../helpers/point"
```

Comment:
__[jshint]__
*  'import' is only available in ES6 (use 'esversion

<!-- Comment by [SideCI](https://sideci.com) -->